pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -48,3 +77,41 @@ func (pc *PackageControllerClient) ValidateControllerDoesNotExist(ctx context.Co
 	}
 	return nil
 }
+
+func (pc *PackageControllerClient) ApplySecret(ctx context.Context) error {
+	eksaAccessKeyId, eksaSecretAccessKey := os.Getenv(config.EksaAccessKeyIdEnv), os.Getenv(config.EksaSecretAcessKeyEnv)
+	eksaRegion := eksaDefaultRegion
+	if region, found := os.LookupEnv(config.EksaRegionEnv); found {
+		eksaRegion = region
+	}
+
+	templateValues := map[string]string{
+		"eksaAccessKeyId":     base64.StdEncoding.EncodeToString([]byte(eksaAccessKeyId)),
+		"eksaSecretAccessKey": base64.StdEncoding.EncodeToString([]byte(eksaSecretAccessKey)),
+		"eksaRegion":          base64.StdEncoding.EncodeToString([]byte(eksaRegion)),
+	}
+
+	result, err := templater.Execute(awsSecretYaml, templateValues)
+	if err != nil {
+		return fmt.Errorf("replacing template values %v", err)
+	}
+
+	params := []string{"create", "-f", "-", "--kubeconfig", pc.kubeConfig}
+	stdOut, err := pc.kubectl.CreateFromYaml(ctx, result, params...)
+	if err != nil {
+		return fmt.Errorf("creating secret %v", err)
+	}
+
+	fmt.Print(&stdOut)
```

Comment:
nit: any specific reason to use `Print` vs `Println`? If the text you print doesn't end with a line break, this will mess the terminal a bit

Reply:
Yes the result from kubectl already sends the result with a new line character. I didn't see the necessity to add an extra /ln to it.